### DISCUSSION TITLE
关于阶段三显示攻击范围和效果的一些要求

### DISCUSSION THREAD

**Discussion by myalice-forever** [liusy58/NJU_AdvancedProgramming_SP22#199] [Q&A]
请问是否需要**全部实现** **敌方角色**和**我方角色**的攻击范围和攻击与被攻击的效果，还是只需要实现一方即可。pdf中的阶段二与阶段三的可选要求（比如buff）需要实现多少，如果没有实现分数会有折扣吗？最后关于拓展功能有何量化要求？

**Comment by liusy58** [comment]
手册/slides有写，请仔细阅读
